plugins/become/machinectl.py

```diff
@@ -66,15 +66,37 @@
             ini:
               - section: machinectl_become_plugin
                 key: password
+    notes:
+      - This plugin requires a polkit rule which will alter the behaviour of machinectl to ask directly for the user
+        credentials if the user is allowed to perform the action
+      - >
+        An example for such a polkit role would be C(
+        polkit.addRule(function(action, subject) {
+          if(action.id == "org.freedesktop.machine1.host-shell" && subject.isInGroup("wheel")) {
```

Comment:
You cannot put complex code into `C()`, since it will terminate at the first `)` it finds. Also it will not preserve more complex spacing. Better put this into the examples as a block comment.